packages/babel/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 [these people](https://github.com/rollup/rollup-plugin-babel/graphs/contributors)
```

Comment:
what should be put here now? 🤔

Reply:
Since pnpm uses the license from the root of the repo when publishing a package, we can safely remove this file. But you raise a good question. I would add a section to the `## Meta` portion of the `babel` plugin README (perhaps `### Prior Contributors`?) that stated that those contributors were responsible for the previous version in the old repo.

Reply:
Hm, thanks to migrated git history and https://github.blog/changelog/2019-12-19-improved-attribution-when-squashing-commits/ we might not need to do that. But let's see what happens once we merge this in.